### PR TITLE
New/Fix update contact in list sendinbleu

### PR DESCRIPTION
**Pull request by atm-john**
- FIX : Add contact to existing list *29/09/2021* - 2.3.1
- NEW : Prepare doc for hooks *29/09/2021* - 2.3.0

### CODE REVIEW

File: langs/fr_FR/sendinblue.lang [hunk 1]
```diff
@@ -71,6 +71,11 @@ SENDINBLUE_EXTRAFIELDS_SOCIETE_ALLOWED=Champs supplémentaires société pour sy
 SENDINBLUE_EXTRAFIELDS_CONTACT_ALLOWED=Champs supplémentaires contact pour synchronisation
 SENDINBLUE_EXTRAFIELDS_SOCIETE_ALLOWEDHelp=Sélectionnez ici les attributs supplémentaires que vous souhaitez synchoniser<br />Pour que cela fonctionne correctement vous devez créer ce/ces même attributs côté Sendinblue dans la séction "Paramètre > Attributs des contacts et CRM > Champs attributs" en utilisant le code de l'attribut Dolibarr comme nom de l'attribut sur Sendinblue
 SENDINBLUE_EXTRAFIELDS_CONTACT_ALLOWEDHelp=Sélectionnez ici les attributs supplémentaires que vous souhaitez synchoniser<br />Pour que cela fonctionne correctement vous devez créer ce/ces même attributs côté Sendinblue dans la séction "Paramètre > Attributs des contacts et CRM > Champs attributs" en utilisant le code de l'attribut Dolibarr comme nom de l'attribut sur Sendinblue
+SENINBLUE_USER_ID = Utilisateur utilisé lors des appels webhook de sendinblue
+SENINBLUE_WebhookUrl = Url des webhook
```
Comment (atm-florianm) [inline langs/fr_FR/sendinblue.lang:75]:
```suggestion
SENINBLUE_WebhookUrl = Url des webhooks
```

File: langs/fr_FR/sendinblue.lang [hunk 2]
```diff
@@ -71,6 +71,11 @@ SENDINBLUE_EXTRAFIELDS_SOCIETE_ALLOWED=Champs supplémentaires société pour sy
...
+SENINBLUE_missingTokenGenerated = La Clé de sécurité pour l'URL de lancement des travaux programmés n'a pas été généré, elle est à générer avec le module tâches planifiées
```
Comment (atm-florianm) [inline langs/fr_FR/sendinblue.lang:76]:
```suggestion
SENINBLUE_missingTokenGenerated = La Clé de sécurité pour l'URL de lancement des travaux programmés n'a pas été générée, elle est à générer avec le module tâches planifiées
```

**Review by atm-florianm — APPROVED** [review]

File: langs/fr_FR/sendinblue.lang (hunk 1 above)
Comment (atm-john) [reply langs/fr_FR/sendinblue.lang:75]: normalement non mais tu as raison

File: admin/sendinblue_setup.php [hunk 3]
```diff
@@ -175,6 +198,37 @@
...
+print '<td>' . $langs->trans("Valeur") . '</td>';
+print '<td></td>';
+print "</tr>\n";
+
+
+
+
+print '<tr class="impair"><td>' . $langs->trans("SendInBlueWebHooksUrl").'</td>';
+print '<td >';
+print '<input style="min-width:100%;" onClick="this.select();" value="'.dol_buildpath('sendinblue/webhook.php', 2).'?token='.$conf->global->CRON_KEY.'" readonly="readonly"  />';
```
[the remark this answers is not in the document]
Comment (atm-john) [reply admin/sendinblue_setup.php]: normalement non mais tu as raison

**Pull request merged**